sections/faqs/extend-and-styled-difference.js

```diff
@@ -0,0 +1,25 @@
+import md from 'components/md'
+
+const CSSFrameworks = () => md`
+  ## What is the difference between \`Comp.extend\` and \`styled(Comp)\`?
+
+
+  You have two choices to add additional styles to a component or a styled component,
+  one is using [\`Comp.extend\`](/docs/basics#extending-styles) and the other is wrapping your
+  component in the styled method like this: [\`styled(Comp)\`](/docs/basics#styling-any-components).
+
+  The styled method works perfectly on all of your own or any third-party components
```

Comment:
At this point I'd just call it a "factory function", but not a method?

- Code highlighting missing